src/components/Loader/Loader.scss

```diff
@@ -11,7 +11,7 @@
     margin: 1em auto;
     border: 0.3em solid #ddd;
     border-left-color: #000;
-    animation: load8 1.2s infinite linear;
+    animation: load8 4s infinite linear;
```

Comment:
?

Reply:
I guess this change, produce this bug:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/182032176-bdf9ef37-cf5a-4159-800d-5aafe4ddd9b1.png">